scripts/check-all-the-things.sh

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env sh
 
-lerna clean --yes \
-&& lerna bootstrap \
-&& lerna run clean --stream \
-&& lerna run compile --stream \
-&& lerna run lint:fix --stream \
-&& lerna run lint:check --stream \
-&& lerna run test --stream
+lerna run format --stream
```

Comment:
I believe that all these steps must be run. `lerna run format` replaces all these steps?